data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostAttesterSlashing.java

```diff
@@ -74,8 +75,8 @@ public void handle(final Context ctx) throws Exception {
           jsonProvider.jsonToObject(ctx.body(), AttesterSlashing.class);
       InternalValidationResult result =
           nodeDataProvider.postAttesterSlashing(attesterSlashing).join();
-      if (result.equals(InternalValidationResult.IGNORE)
-          || result.equals(InternalValidationResult.REJECT)) {
+      if (result.code().equals(ValidationResultCode.IGNORE)
+          || result.code().equals(ValidationResultCode.REJECT)) {
```

Comment:
I'd be very tempted to have `isIgnore()`, `isReject()`, `isAccept()` etc type methods on `InternalValidationResult` to make this simpler.  Almost tempted to have `isIgnoreOrReject()` if I could think of a better name for it (e.g. `isNotProcessable()` or something...)